setup.py

```diff
@@ -67,6 +67,7 @@ def enable_log_to_stdout(logname):
         str(plugin_name),  # https://github.com/pypa/setuptools/pull/597
     ],
     package_dir={plugin_name: plugin_name},
+    install_requires=["smpplib2==0.1.1"],
```

Comment:
@aronasorman @rtibbles Is it OK to include a requirement within a plugin like this? Seems to work for local dev at least.